### PR TITLE
fix #11, new table

### LINKED ISSUE #11: Schedule App not using CFE Table
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Schedule lab App is not using cFE table service. Recommend implementing cFE table service.

**Describe alternatives you've considered**
None

**Additional context**
none

**Requester Info**
Anh Van, NASA Goddard

### PR DESCRIPTION
**Pull request by avan989**
**Describe the contribution**
fix #11, new table, reference closed pull request #12, reopen of it version of it.  

**Testing performed**
references closed pull request #12, just a reopen version of it. 

**Contributor Info**
Anh Van, NASA Goddard

**Community contributors**
You must attach a signed CLA (required for acceptance) or reference one already submitted

### CODE REVIEW

File: fsw/src/sch_lab_app.c [hunk 1]
```diff
@@ -137,6 +122,124 @@ void SCH_Lab_AppMain(void)
...
+    CFE_ES_ExitApp( Status );
     
 }/* end SCH_Lab_AppMain */
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */ 
+/*                                                                 */ 
+/* AppInit() -- initialization                                     */ 
+/*                                                                 */ 
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+int32 AppInit(void)
```
Comment (skliper) [inline fsw/src/sch_lab_app.c]: SCH_Lab_AppInit

File: fsw/platform_inc/sch_lab_sched_tab.h [hunk 2]
```diff
@@ -53,49 +53,23 @@
...
-     { FM_SEND_HK_MID,       4, 0 },
-     { DS_SEND_HK_MID,       4, 0 },
-     { LC_SEND_HK_MID,       4, 0 },
-
-#endif
-
-     { SCH_LAB_END_OF_TABLE, 0, 0 }
-};
 
+int32 SchTblValidation(void *MySchTBL);
```
Comment (skliper) [inline fsw/platform_inc/sch_lab_sched_tab.h]: Preference is SCH_Lab_TblValidation

File: fsw/src/sch_lab_app.c (hunk 1 above)
Comment (skliper) [reply fsw/src/sch_lab_app.c]: Update reviewed, approved.

File: fsw/platform_inc/sch_lab_sched_tab.h (hunk 2 above)
Comment (skliper) [reply fsw/platform_inc/sch_lab_sched_tab.h]: Update reviewed and approved.

**Comment by avan989** [comment]
updated to use standard name. Compiled and tested telemetry with cosmos. No error

**Pull request merged**